index.js

```diff
@@ -186,6 +190,33 @@ class LedgerBridgeKeyring extends EventEmitter {
     delete this.accountDetails[ethUtil.toChecksumAddress(address)]
   }
 
+  updateTransportMethod(useLedgerLive = false) {
+    return new Promise((resolve, reject) => {
+      // If the iframe isn't loaded yet, let's store the desired useLedgerLive value and
+      // optimistically return a successful promise
+      if(!this.iframeLoaded) {
+        return new Promise((delayedResolve, delayedReject) => {
+          this.delayedPromise = {
+            resolve: delayedResolve,
+            reject: delayedReject,
+            useLedgerLive,
+          };
+        });
```

Comment:
Wouldn't this never resolve in this case, because `resolve` is never called? :thinking: 

I think this Promise constructor is redundant, because the outer one is here already.

```suggestion
        this.delayedPromise = {
          resolve,
          reject,
          useLedgerLive,
        };
        return;
```